From: Dermot Bradley <dermot_bradley@yahoo.com>
Date: Thu, 26 Aug 2021 00:58 +0100
Subject: [PATCH] cloud-init: Prevent duplicate console messages

Prevent duplicate messages for each user (without SSH keys) appearing
on console.

---

diff -aur a/cloudinit/config/cc_ssh_authkey_fingerprints.py b/cloudinit/config/cc_ssh_authkey_fingerprints.py
--- a/cloudinit/config/cc_ssh_authkey_fingerprints.py
+++ b/cloudinit/config/cc_ssh_authkey_fingerprints.py
@@ -70,7 +70,7 @@
     if not key_entries:
         message = ("%sno authorized SSH keys fingerprints found for user %s.\n"
                    % (prefix, user))
-        util.multi_log(message)
+        util.multi_log(message, console=True, stderr=False)
         return
     tbl_fields = ['Keytype', 'Fingerprint (%s)' % (hash_meth), 'Options',
                   'Comment']
